lib/std/os/linux.zig

```diff
@@ -66,6 +66,27 @@ pub const socketcall = syscall_bits.socketcall;
 pub const syscall_pipe = syscall_bits.syscall_pipe;
 pub const syscall_fork = syscall_bits.syscall_fork;
 
+pub fn clone(
+    func: *const fn (arg: usize) callconv(.C) u8,
+    stack: usize,
+    flags: u32,
+    arg: usize,
+    ptid: *i32,
+    tls_: usize,
```

Comment:
This conflicted with the `pub const tls = @import("linux/tls.zig");` in this file. Not sure what the usual convention is in cases like this.

Reply:
I'll just rename it to `tp` (even though the man page calls it `tls`).